Content.Server/SS220/TeleportAFKtoCryoSystem/TeleportAFKtoCryoSystem.cs

```diff
@@ -22,6 +22,8 @@
 using Content.Shared.DoAfter;
 using Content.Shared.SS220.TeleportAFKtoCryoSystem;
 using Content.Shared.Administration.Logs;
+using Content.Server.Ghost; // SS220 Cryostorage ghost role fix begin
+using Content.Server.Mind; // SS220 Cryostorage ghost role fix begin
```

Comment:
```suggestion
using Content.Server.Mind; // SS220 Cryostorage ghost role fix end
```
`SS220` в целом можно и не писать в наших файлах

Reply:
Я даже не увидел что это наши... удалю пометки